### ISSUE TITLE
about the label format

### ISSUE THREAD

**Issue by ShowTsai** [PavlosMelissinos/enet-keras#20]
I have my own dataset
There are 34774 png input images and 34774 png labels images
input images are in shape (576, 576, 3)
label images are in shape(576, 576)
Every pixel in label image has a class number
and there are 6 classes
I don't quite understand how you deal with the MSCOCO annotations
and what you do in "flow()" function in datasets.py,
So what should I do on the"flow()"function , thanks!

**Comment by PavlosMelissinos** [comment]
Hey, apologies for taking so long to respond.

1.  I remember that the annotation system used by MS-COCO for semantic segmentation is kinda unusual if you're coming from a classification background. However, the gist is that datasets.py internally does some kind of conversion between alpha-like masks and polygons. Unfortunately you'll have to read the code in order to figure out exactly how it works, sorry about that :(
2. flow is a function that lazily iterates over the dataset (it 'flows' over it) and is similar to `keras.preprocessing.image.ImageDataGenerator.flow`.

Other than that, is this issue still relevant to you or anyone else?

**Comment by PavlosMelissinos** [comment]
> Other than that, is this issue still relevant to you or anyone else?

I suppose not... Closing it for the moment
